xde/src/xde.rs

```diff
@@ -1758,6 +1760,7 @@ fn next_hop<'a>(
             );
             return (EtherAddr::zero(), EtherAddr::zero(), underlay_port);
         }
+        ip::ire_refrele(gw_ire);
```

Comment:
Looking back at this, it doesn't seem like we actually ever use `gw_ire`? Was that lookup just to make sure such an IRE exists?

Reply:
Yeah, this is just a check to make sure the link-local address of the next hop has an IRE, and we bail early if that is not the case.